README.md

```diff
@@ -1,2 +1,34 @@
 # altinn-components
-Reusable react components
+This package is a set of reusable components for building web applications for [Altinn 3](https://docs.altinn.studio/nb/community/about/).
+This is *not* a general purpose library, but a library of components that are specifically designed for the Altinn platform, to ensure
+a consistent look and feel across all applications. It also contains a storybook with stories explain the purpose and usage of each component, and
```

Comment:
(...) It also includes a storybook with stories explaining the purpose and usage of each component, with real world examples of how to use them separately or together (composition).